socorro/external/es/supersearch.py

```diff
@@ -505,12 +523,30 @@ def get(self, **kwargs):
                     hits = []
                     total = 0
                     aggregations = {}
+                    shards = None
                     break
 
+        if shards and shards.failed:
+            # Some shards failed. We want to explain what happened in the
+            # results, so the client can decide what to do.
+            failed_indices = defaultdict(int)
+            for failure in shards.failures:
+                failed_indices[failure.index] += 1
+
+            for index, shards_count in failed_indices.items():
+                errors.append({
+                    'type': 'shards',
+                    'index': index,
+                    'shards_count': shards_count,
+                })
+
+            # TODO: send an error to sentry?
```

Comment:
Nope. We only want to send to sentry when there's a exception "frame". E.g. something in `sys.exc_info()`.